nautobot/docs/installation/upgrading-from-nautobot-v1.md

```diff
@@ -310,7 +329,6 @@ Their filters are also being replaced by `?location=<uuid/slug>`. For example `/
 
 | Model              | Removed Filter Field  | UI and API endpoints that are no longer supported in v2.X                                     |
 |--------------------|-----------------------|-----------------------------------------------------------------------------------------------|
-| Aggregate          | `tenant_group_id`     |                                                                                               |
```

Comment:
Probably need to document removal of `Tenant` `aggregates` and `has_aggregates` filters here.